src/reference/config/telemetry/exporters/exporter-syslog.md

```diff
@@ -0,0 +1,130 @@
+---
+shortTitle: syslog
+description: Zilla runtime syslog exporter
+category:
+  - Telemetry
+tag:
+  - Exporters
+---
+
+[Available in <ZillaPlus/>](https://www.aklivity.io/products/zilla-plus)
+{.zilla-plus-badge .hint-container .info}
+
+# Syslog Exporter
+
+Zilla runtime Syslog exporter
+
+```yaml {3}
+exporters:
+  syslog:
+    type: syslog
+    options:
+      host: syslog-server
+      port: 514
+      protocol: tcp
+```
+
+with vault:
+```yaml {11}
+vaults:
+  vault:
+    type: filesystem
+    options:
+      trust:
+        store: truststore.p12
+        type: pkcs12
+        password: generated
+exporters:
+  syslog:
+    type: syslog
+    vault: vault
+    options:
+      host: syslog-server
+      port: 6514
+      protocol: tls
+      trust:
+        - syslog
+```
+
+## Configuration
+
+:::: note Properties
+
+- [options\*](#options)
+  - [options.host\*](#options-host)
+  - [options.port\*](#options-port)
+  - [options.protocol\*](#options-protocol)
+  - [options.trust](#options-trust)
+  - [options.trustcacerts](#options-trustcacerts)
+
+::: right
+\* required
+:::
+
+::::
+
+### options
+
+> `object`
+
+`syslog`-specific options.
+
+```yaml {4}
+options:
+  host: syslog-server
+  port: 514
+  protocol: tcp
+```
+
+```yaml {4}
+options:
+  host: syslog-server
+  port: 514
+  protocol: udp
+```
+
+```yaml {4}
+options:
+  host: syslog-server
+  port: 6514
+  protocol: tls
+  trust:
+    - syslog
+```
+
+#### options.host
+
+> `string`
+
+The hostname of the syslog server.
+
+#### options.port
+
+> `integer`
+
+The port of the syslog server.
+
+#### options.protocol
+
+> `enum` [ "tcp", "udp", "tls" ] | Default: `"tcp"`
+
+The protocol to use to communicate with the syslog server. Valid values are: `tcp`, `udp`, `tls`.
+
+#### options.trust
+
+> `array` of `string`
+
+Keys in the vault referenced on the binding (e.g. a filesystem vault for a local pkcs12 keystore
+or an AWS vault for remote pem format certificates stored in AWS secrets manager). Only valid if the protocol is `tls`.
```

Comment:
Let's use the tls binding as a guide on how to document this.
https://github.com/aklivity/zilla-docs/blob/develop/src/reference/config/bindings/binding-tls.md

The first example probably needs to include a vault name and the tls configuration.

Reply:
I added the missing doc entry for the vault param.

I have to examples on the top of the page, one without vault, and one with vault, and tls like this
```yaml {3}
exporters:
  syslog:
    type: syslog
    options:
      host: syslog-server
      port: 514
      protocol: tcp
```

with vault:
```
vaults:
  vault:
    type: filesystem
    options:
      trust:
        store: truststore.p12
        type: pkcs12
        password: generated
exporters:
  syslog:
    type: syslog
    vault: vault
    options:
      host: syslog-server
      port: 6514
      protocol: tls
      trust:
        - syslog
```

Pls LMK if this is OK or if we need something more.